File: RepoTransBench/target_projects/Python/C++/aadhithya_rajiniPP/public_tests/test_public_rajiniworld.cpp
#include <gtest/gtest.h>
#include "../src/rajinipp_rajiniworld.h"

TEST(PublicRajiniworldTest, VarsAndFunctionsDictNonempty) {
    auto& vars = rajinipp_rajiniworld::__vars__;
    auto& funcs = rajinipp_rajiniworld::__functions__;
    EXPECT_TRUE(vars.is_map());
    EXPECT_TRUE(funcs.is_map());
    EXPECT_TRUE(&vars != nullptr);
    EXPECT_TRUE(&funcs != nullptr);
}

File: RepoTransBench/target_projects/Python/Rust/aadhithya_rajiniPP/public_tests/test_public_rajiniworld.rs
use aadhithya_rajiniPP::__rajiniworld__::{__vars__, __functions__};
use std::collections::HashMap;

#[test]
fn test_public_vars_and_functions_dict_nonempty() {
    let _: &HashMap<&'static str, f64> = &__vars__;
    let _: &HashMap<&'static str, fn()> = &__functions__;
}